src/components/InstrumentPicker/index.tsx

```diff
@@ -67,10 +67,12 @@ const connectedInstrumentPicker = connect(
       referenceStore.getAssetById(referenceStore.baseAssetId) ||
       new AssetModel({}),
     instrumentId: pathOr(undefined, ['selectedInstrument', 'id'], uiStore),
-    instruments: referenceStore.findInstruments(
-      uiStore.searchTerm,
-      uiStore.searchWalletName
-    ),
+    instruments: referenceStore
+      .findInstruments(uiStore.searchTerm, uiStore.searchWalletName)
+      .sort(
+        (a: InstrumentModel, b: InstrumentModel) =>
```

Comment:
you don't need type annotation here, it should be inferred by tsc, if list is not empty